libcloud/compute/drivers/ec2.py

```diff
@@ -3906,6 +3924,138 @@ def ex_delete_placement_group(self, name):
         response = self.connection.request(self.path, params=params).object
         return self._get_boolean(response)
 
+    def ex_import_snapshot(self, client_data=None,
+                           client_token=None, description=None,
+                           disk_container=None, dry_run=None, role_name=None):
+        """
+        Imports a disk into an EBS snapshot. More information can be found
+        at https://goo.gl/sbXkYA.
+
+        :param  client_data: Describes the client specific data (optional)
+        :type   client_data: ''dict''
```

Comment:
these should be backticks, instead of single quotes. I'm not sure RST would mark them as an internal type otherwise.